src/main/java/org/jenkinsci/plugins/ghprb/upstream/GhprbUpstreamStatusListener.java

```diff
@@ -59,15 +60,15 @@ private boolean updateEnvironmentVars(AbstractBuild<?, ?> build, TaskListener li
             statusMessages.add(new GhprbBuildResultMessage(state, message));
         }
         
-        String context = envVars.get("commitStatusContext");
+        String context = envVars.get("ghprbCommitStatusContext");
         
         if (StringUtils.isEmpty(context)) {
             context = jobName;
         }
         
-        Boolean addTestResults = new Boolean(envVars.get("ghprbStartedStatus"));
+        Boolean addTestResults = Boolean.valueOf(envVars.get("ghprbStartedStatus"));
 
-        statusUpdater = new GhprbSimpleStatus(envVars.get("ghprbCommitStatusContext"), envVars.get("ghprbStatusUrl"), envVars.get("ghprbTriggeredStatus"), envVars.get("ghprbStartedStatus"), addTestResults, statusMessages);
+        statusUpdater = new GhprbSimpleStatus(context, envVars.get("ghprbStatusUrl"), envVars.get("ghprbTriggeredStatus"), envVars.get("ghprbStartedStatus"), addTestResults, statusMessages);
```

Comment:
This was a bug too, the context value above was being set to the job name if it was empty, but then not being used.